interfaces/cython/cantera/composite.py

```diff
@@ -1226,11 +1235,11 @@ def write_hdf(self, filename, *args, cols=None, group=None, subgroup=None,
         :return:
             Group identifier used for storing HDF data.
 
-        Arguments *compression*, and *compression_opts* are mapped to parameters
+        Arguments ``compression``, and ``compression_opts`` are mapped to parameters
         for `h5py.create_dataset`; in both cases, the choices of `None` results
```

Comment:
Does this create link to this method?
```suggestion
        for `h5py.create_dataset`; in both cases, the choices of ``None`` results
```